q2_moshpit/plugin_setup.py

```diff
@@ -379,6 +379,25 @@
                 'analyses.'
 )
 
+plugin.methods.register_function(
+    function=q2_moshpit.eggnog.fetch_eggnog_db,
+    inputs={},
+    parameters={},
+    outputs=[("eggnog_db", ReferenceDB[Eggnog])],
+    output_descriptions={
+        "eggnog_db": "Artifact containing the eggNOG annotation "
+                     "database (eggnog.db, eggnog.taxa.db, "
+                     "eggnog.taxa.db.traverse.pkl)"
+    },
+    name="Fetch one of the databases necessary to run eggNOG.",
```

Comment:
Maybe we can be a bit more specific and say "to run the eggnog-annotate action"? With all those databases it is becoming confusing which is needed where 😅